### DISCUSSION TITLE
HRR.csv unit measure and scientific notation interpretation

### DISCUSSION THREAD

**Discussion by LordMicidial** [firemodels/fds#13048] [Q&A] [answered]
I've recently noticed the how hrr.csv stores data. I tried looking at the User Guide, the Google Group and here but I've found nothing so far.

Results are compatible with what I expected in terms of shape of the HRR curve.

Anyway I always get huge number noted down with scientific notation (E+). I can't understand why it happens. Also, the User Guide (pag. 492) clearly shows that first column should indicate time in seconds.

![image](https://github.com/firemodels/fds/assets/165648069/c0ca5c9c-caf1-459a-8848-15e524d37649)

As per the image, time starts with 6,74E+06. This is hard to grasp to me. Simulation starts should end at 400 s. Also HRR values are extremely big, but I won't enter in such details given the fact that computation might be hard to investigate backwards but time should be easier.

I'm using FDS 6.9, but I've seen the same issue on other simulation run on other versions.

Thanks!

here below the extremely big values of HRR

![image](https://github.com/firemodels/fds/assets/165648069/36e8baac-f56f-4434-a8b1-f94d29569456)

**Comment by drjfloyd** [comment, marked as answer]
FDS writes numbers in the US customary format where the comma is used to seperate values in the csv and a period "." is used to seperate integer from decimal values in the number (e.g. six fifths written as 1.2) .  You appear to have your spreadsheet set to use the comma to seperate integer from decimal values (e.g. six fifths written as 1,2).
